docs/sources/contribute/release-notes/index.md

```diff
@@ -128,7 +128,11 @@ When you’re ready to add a What’s new entry, complete the following steps:
        <tr>
          <th scope="row">ENABLEMENT VIDEO (OPTIONAL)</th>
          <td>Link to the video used for enablement.</td>
-         <td></td>
+         <td>
+           <p>Enablement videos are perhaps the fastest, most engaging tool for employees and users to learn about your feature. Use them for maximum engagement.</p>
+           <p>Follow these instructions to create and upload a video: <a href="https://docs.google.com/document/d/1nCiG62FxJ9J_qLTnlzNopSsT-VEAlxCUCsrAEpWL68U/edit#heading=h.fierz9i4q8ft">Enablement video instructions</a>.</p>
+           <p>When you upload an enablement video, the Content team receives a notification, edits it for the public, and uploads it to YouTube to coincide with your feature's release. They need a few weeks' lead time for this.</p>
```

Comment:
My only nit is that "a few weeks" is kinda squishy. I'd say let's suggest a hard minimum time frame here.